### ISSUE TITLE
Can I use a native solc compiler with slither?

### ISSUE THREAD

**Issue by chebykin** [crytic/slither#268]
I'm getting the following error when running `slither .` inside my truffle project root folder:

```
ERROR:root:Error in .
ERROR:root:Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/slither/__main__.py", line 569, in main_impl
    (results, number_contracts) = process(filename, args, detector_classes, printer_classes)
  File "/usr/local/lib/python3.7/site-packages/slither/__main__.py", line 51, in process
    **vars(args))
  File "/usr/local/lib/python3.7/site-packages/slither/slither.py", line 55, in __init__
    crytic_compile = CryticCompile(contract, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/crytic_compile/crytic_compile.py", line 68, in __init__
    self._compile(target, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/crytic_compile/crytic_compile.py", line 590, in _compile
    self._platform.compile(self, target, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/crytic_compile/platform/truffle.py", line 53, in compile
    version, compiler = _get_version(base_cmd, cwd=target)
  File "/usr/local/lib/python3.7/site-packages/crytic_compile/platform/truffle.py", line 145, in _get_version
    version = re.findall('\d+\.\d+\.\d+', line)[0]
IndexError: list index out of range
```

I use a native compiler, specified in `truffle.json` config file with:

```js
  ...
  networks: {
  },
  compilers: {
    solc: {
      version: 'native',
      settings: {
        optimizer: {
          enabled: true,
          runs: 200
        }
      }
    }
  }
  ...
```

It seems that `slither` can't properly detect `solc` version in my case. Is it supposed to be compatible with native `solc` binary, not `js` one?

**Comment by montyly** [comment]
Hi @chebykin ,

Thank you for reporting this, it was an issue in our [`crytic-compile`](https://github.com/crytic/crytic-compile) library. We added the support in https://github.com/crytic/crytic-compile/pull/20, and we are going to release a new version containing the fix soon (in a few days).

In the meantime, if you use Slither with [pyenv](https://github.com/crytic/slither/wiki/Developer-installation), you can use the latest git version of crytic-compile: 
```
git clone https://github.com/crytic/crytic-compile
cd crytic-compile
python setup.py develop
```

**Comment by montyly** [comment]
Hi @chebykin 

We released `crytic-compile` [0.1.2](https://github.com/crytic/crytic-compile/releases/tag/v0.1.2). If you update your version `pip install crytic-compile --upgrade`, you will be able to use Slither with truffle and native solc.

Let us know if you have any issue

**Comment by montyly** [comment]
Hi @chebykin,

As I think it was fixed with `crytic-compile` 0.1.2, I am closing the issue. Let us know if you have any troubles!
